utils/spacewalk-common-channels.ini

```
@@ -1241,6 +1321,26 @@ gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/SLE15-Uyuni-Client-Tools/SLE_15/
 
+[sles15-sap-sp3-uyuni-client]
+name     = Uyuni Client Tools for SLES15 SP3 %(arch)s
+archs    =  x86_64, s390x, aarch64, ppc64le
+base_channels = sle-product-sles_sap15-sp3-pool-%(arch)s
+checksum = sha256
+gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/SLE15-Uyuni-Client-Tools/SLE_15/repodata/repomd.xml.key
+gpgkey_id = %(_uyuni_gpgkey_id)s
+gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
+repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/SLE15-Uyuni-Client-Tools/SLE_15/
+
+[sles15-sap-sp3-devel-uyuni-client]
+name     = Uyuni Client Tools for SLES15 SP3 %(arch)s (Development)
+archs    =  x86_64, s390x, aarch64, ppc64le
+base_channels = sle-product-sles_sap15-sp3-pool-%(arch)s
+checksum = sha256
+gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/SLE15-Uyuni-Client-Tools/SLE_15/repodata/repomd.xml.key
```

```suggestion
gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/SLE15-Uyuni-Client-Tools/SLE_15/repodata/repomd.xml.key
```

(as this is for the devel client tools)